### PR TITLE
feat: DHIS2-9276 #8 - hide empty menus

### PR DESCRIPTION
**Pull request by JoakimSM**
[DHIS2-9276](https://jira.dhis2.org/browse/DHIS2-9276)

Pretty straight forward one, hides the more menu (three dots icon) for the list itself and for each row.

### CODE REVIEW

File: src/core_modules/capture-core/components/ListView/Main/ListViewMain.component.js [hunk 1]
```diff
@@ -80,11 +80,18 @@ class ListViewMainPlain extends React.PureComponent<Props> {
             filters,
             updatingWithDialog,
             onSelectRow,
+            customRowMenuContents,
             ...passOnProps
         } = this.props;
+
+        const ListComponent = (!customRowMenuContents || customRowMenuContents.length <= 0) ?
```
Comment (paschalidi) [inline src/core_modules/capture-core/components/ListView/Main/ListViewMain.component.js:87]: I think in this case we can say or could an array length be `< 0`? 
```suggestion
        const ListComponent = (!customRowMenuContents || !customRowMenuContents.length) ?
```

File: src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.js [hunk 2]
```diff
@@ -88,6 +88,10 @@ const ListViewMenuPlain = ({ customMenuContents = [], classes }: Props) => {
         </Paper>
     ), [renderMenuItems]);
 
+    if (customMenuContents.length <= 0) {
```
Comment (paschalidi) [inline src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.js:91]: similar question/suggestion

**Review by paschalidi — COMMENTED** [review]
Hey looks great, one small comment

File: src/core_modules/capture-core/components/ListView/Main/ListViewMain.component.js (hunk 1 above)
Comment (JoakimSM) [reply src/core_modules/capture-core/components/ListView/Main/ListViewMain.component.js:87]: Changing. Old habbit.

File: src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.js (hunk 2 above)
Comment (JoakimSM) [reply src/core_modules/capture-core/components/ListView/Menu/ListViewMenu.component.js:91]: Changing. Old habbit.

**Review by paschalidi — APPROVED** [review]
thanks for the changes on this one as well 👍

**Pull request merged**